lib/grape/path.rb

```diff
@@ -22,11 +22,11 @@ def root_prefix
     end
 
     def uses_specific_format?
-      !!(settings[:format] && Array(settings[:content_types]).size == 1)
+      (settings[:format] && Array(settings[:content_types]).size == 1)
     end
 
     def uses_path_versioning?
-      !!(settings[:version] && settings[:version_options][:using] == :path)
+      (settings[:version] && settings[:version_options][:using] == :path)
```

Comment:
@dblock 
The error in the specs is related to this line change.
My problem is with the way ruby style-guide/rubocop addresses this violation.

```
# bad
!!(settings[:format] && Array(settings[:content_types]).size == 1)

#good
!(settings[:format] && Array(settings[:content_types]).size == 1).nil?
```

This raises the problem of if this operation returns `false`, it will evaluate to true, which makes it return the incorrect value.
I personally am not opposed to `!!` and think it is in fact better than the alternative `!foo.nil?`
Potentially, I could rework this PR to simply disable this check and move it to `.rubocop.yml` as disabled

Thoughts?